### PR TITLE
update oil and gas fields info

### PR DESCRIPTION
**Pull request by gregbunce**
update new feature service name and add prod codes

### CODE REVIEW

File: data/energy/oil-gas/index.html [hunk 1]
```diff
@@ -221,7 +221,60 @@ <h3 id="OilandGasFields">Oil and Gas Fields</h3>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{% link images/OilFieldsSM.png %}" alt="Oil Fields" />
     <p>This dataset depicts oil and gas field polygons in Utah from the Utah Division of Oil, Gas and Mining. The dataset contains the field number, field name, status and more.</p>
+    <p>This layer is the most up-to-date version of the oil and gas fields in Utah. In 2020, ten fields were specifically expanded to incorporate wildcat/undesignated wells within the Uinta Basin.</p>
     <p>Contact <a href="https://oilgas.ogm.utah.gov/oilgasweb/contact-us/contact-us-main.xhtml">the Utah Division of Oil, Gas and Mining</a> with any questions.</p>
+    <p>Provided here are the most widely used PROD codes in the dataset. These codes are used in the following fields: PROD_FORM, FORM_2, FORM_4, and FORM_4.</p>  
+    <p>A complete list of PROD codes can be found [here](https://docs.google.com/spreadsheets/d/1O8PRzPuTXY9vhTikZj9G5Y6rmLL6x8p0MCD6nTvwPf8/edit?usp=sharing).</p>
```
Comment (steveoh) [inline data/energy/oil-gas/index.html:227]: **issue**: mixing markdown in an html file

File: data/energy/oil-gas/index.html [hunk 2]
```diff
@@ -17,11 +17,11 @@
     - Nightly
 OilandGasFields:
   hub:
-    name: Utah Oil Gas Fields
+    name: Utah Oil and Gas Fields
```
Comment (steveoh) [inline data/energy/oil-gas/index.html:20]: **praise**: nice catch on the wrong name

**Review by steveoh — COMMENTED** [review]
you'll see the issue in the preview

File: data/energy/oil-gas/index.html (hunk 2 above)
Comment (gregbunce) [reply data/energy/oil-gas/index.html:20]: thanks, that one took me a min.

File: data/energy/oil-gas/index.html (hunk 1 above)
Comment (gregbunce) [reply data/energy/oil-gas/index.html:227]: yup, that's the issue!  thx

**Review by steveoh — APPROVED** [review]

**Pull request merged**